### DISCUSSION TITLE
Metric Calculations using extacted fields of JSON

### DISCUSSION THREAD

**Discussion by thamps** [grafana/grafana-infinity-datasource#23] [General]
Hi,

I am trying to understand if we can do any calculations/aggregations using this plugin. I have created a json source which I have added into grafana and extracting the fields as expected.

Now I am wondering if we can calculate difference between column fields to write field "(value_now - (value_previous))"

ex.
timestamp2 | id | value | (timestamp2 - timestamp1)
timestamp1 | id | value | ..

**Comment by yesoreyeram** [comment]
Calculated fields are not yet available. Use grafana's native transform feature to perform such operations.
